x/config/configurator.go

```diff
@@ -197,30 +197,30 @@ func (c *Configurator) NewDispatcher(serviceName string, data interface{}) (*yar
 
 func (c *Configurator) load(serviceName string, cfg *yarpcConfig) (yarpc.Config, error) {
 	var (
-		errors []error
+		errors error
```

Comment:
I didn't name it `err` because there's always an `err` in scope when appending to this.